api/add_dimension.md

```diff
@@ -11,79 +11,52 @@ api:
-`avg(temperature) FROM conditions GROUP BY hour, location`
-when using `location` as a hash partition). Please see our
-[best practices about partitioning in distributed hypertables][distributed-hypertable-partitioning-best-practices]
-for more information.
-
-### Parallelizing disk I/O on a single node
-
-Parallel I/O can benefit in two scenarios: (a) two or more concurrent
-queries should be able to read from different disks in parallel, or
-(b) a single query should be able to use query parallelization to read
-from multiple disks in parallel.
-
-Thus, users looking for parallel I/O have two options:
-
-1.  Use a RAID setup across multiple physical disks, and expose a
-single logical disk to the hypertable (that is, via a single tablespace).
-
-1.  For each physical disk, add a separate tablespace to the
-database. Timescale allows you to actually add multiple tablespaces
-to a *single* hypertable (although under the covers, a hypertable's
-chunks are spread across the tablespaces associated with that hypertable).
-
-We recommend a RAID setup when possible, as it supports both forms of
-parallelization described above (that is, separate queries to separate
-disks, single query to multiple disks in parallel).  The multiple
-tablespace approach only supports the former. With a RAID setup,
-*no spatial partitioning is required*.
-
-That said, when using hash partitions, we recommend using 1
-hash partition per disk.
-
-Timescale does *not* benefit from a very large number of hash
-partitions (such as the number of unique items you expect in partition
-field).  A very large number of such partitions leads both to poorer
+Every distinct item in hash partitioning is hashed to one of *N* buckets. By default, 
+TimescaleDB uses flexible range intervals to manage chunk sizes. The main purpose of hash
+partitioning is to enable parallelization across multiple disks within the same time 
+interval.
+
+### Parallelizing disk I/O 
+
+You use Parallel I/O in the following scenarios:
+
+- Two or more concurrent queries should be able to read from different disks in parallel.
+- A single query should be able to use query parallelization to read from multiple disks in parallel.
+
+For the following options:
+
+- **RAID**: use a RAID setup across multiple physical disks, and expose a single logical disk to the hypertable.
+  That is, using a single tablespace.
+
+  Best practice is to use RAID when possible. This is because it supports the concurrent and single
+  scenarios and *no spatial partitioning is required*.
+
+- **Multiple tablespaces**: for each physical disk, add a separate tablespace to the database. TimescaleDB allows you to add 
+  multiple tablespaces to a *single* hypertable. However, although under the hood, a hypertable's
+  chunks are spread across the tablespaces associated with that hypertable.
+
+  Multiple tablespaces only supports concurrent queries.
+
+When using hash partitions, best practice is to use 1 hash partition per disk.
```

Comment:
I am not sure we have a best practice for this. But I guess it is reasonably to have _at least_ 1 partition per disk. Personally, I would probably say it is better to have number of partitions set to a multiple of number of disks, e.g., the number of partitions P=N*Pd where N is the number of disks and Pd is the number of partitions per disk. Then you could also add more disks later and move partitions to the new disk from other disks. But maybe this is too much information for this section?

Reply:
Updated.